srv/service.js

```diff
@@ -1,14 +1,11 @@
 // REVISIT: cds.OutboxService or technique to avoid extending OutboxService
 const OutboxService = require('@sap/cds/libx/_runtime/messaging/Outbox');
+const { buildNotification } = require("./../lib/utils");
 
 module.exports = class NotificationService extends OutboxService {
```

Comment:
```suggestion
const { buildNotification } = require("./../lib/utils");

const cds = require('@sap/cds');
const Base = cds.outboxed ? cds.Service : require('@sap/cds/libx/_runtime/messaging/Outbox');

module.exports = class NotificationService extends Base {
```
hybrid outboxing (cf. https://github.com/cap-js/audit-logging/blob/main/srv/service.js#L3-L5)